source/drivers/php-libraries.txt

```diff
@@ -18,6 +18,8 @@ Libraries for the ``mongodb`` Extension
 Stand-alone Libraries
 ~~~~~~~~~~~~~~~~~~~~~
 
+- `Xenus <https://github.com/abellion/xenus>`_ is an extendable, easy to use MongoDB ODM supporting relationships, embedded documents and more.
```

Comment:
```suggestion
- `Xenus <https://github.com/abellion/xenus>`_ is an extendable and easy to use
   MongoDB ODM that supports relationships, embedded documents, and more.
```

Added word wrap and Oxford comma. I think this entry should be added at the bottom of the list, both for alphabetical reasons and due to the project size (mix of Packagist download stats and GitHub activity).

Reply:
Sounds good 👍